templates/primary-secondary/primary-secondary.js

```diff
@@ -701,26 +698,22 @@ class TemplatePrimarySecondary extends RtlMixin(LocalizeCoreElement(LitElement))
 				cursor: ew-resize;
 				width: 0.45rem;
 			}
-			:host([resizable]) [data-is-collapsed] .d2l-template-primary-secondary-divider,
-			:host([resizable][dir="rtl"]) [data-is-expanded] .d2l-template-primary-secondary-divider {
-				cursor: w-resize;
+			:host([resizable]) [data-is-expanded] .d2l-template-primary-secondary-divider {
+				cursor: var(--d2l-cursor-resize-inline-end, e-resize);
 			}
-			:host([resizable]) [data-is-expanded] .d2l-template-primary-secondary-divider,
-			:host([resizable][dir="rtl"]) [data-is-collapsed] .d2l-template-primary-secondary-divider {
-				cursor: e-resize;
+			:host([resizable]) [data-is-collapsed] .d2l-template-primary-secondary-divider {
+				cursor: var(--d2l-cursor-resize-inline-start, w-resize);
 			}
 			:host([resizable]) .d2l-template-primary-secondary-divider-handle {
 				align-items: center;
 				display: flex;
 				justify-content: center;
 			}
-			:host([resizable]) [data-background-shading="secondary"] .d2l-template-primary-secondary-divider,
```

Comment:
This dimension difference was a little annoying to deal with. The CSS variable solve the issue without changing how the style is achieved.